extension/src/popup/components/Onboarding/index.tsx

```diff
@@ -7,18 +7,67 @@ import "./styles.scss";
 
 export const Onboarding = ({
   goBack,
+  hasGoBackBtn,
   children,
 }: {
   goBack?: () => void;
```

Comment:
💭  could we add a comment here saying something like 
`// if null defaults to the BackButton goback`

took me a minute to figure out that was happening (and why both a hasGoBackBtn and a goBack var are needed)

Reply:
Good callout. This is not clear. I'll fix this